state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -64,6 +64,9 @@ use sp_std::{
 };
 pub use weights::WeightInfo;
 
+// 1 hour.
+const TAINTED_TX_EXPIRATION_BLOCKS: u32 = 600;
```

Comment:
Should we tie this up to `SECONDS_PER_BLOCK` in case we do end up changin it? 😄

Reply:
We will not change it :D. But yes, using `SECONDS_PER_BLOCK` is better 👍